rcode-reader/src/main/java/eu/rssw/pct/elements/DataSourceElement.java

```diff
@@ -0,0 +1,63 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.AccessibleClassElement;
+import eu.rssw.pct.RCodeInfo;
+
+public class DataSourceElement extends AccessibleClassElement {
+  private int flags;
+  private int crc;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "crc" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1068)